packages.yaml

```diff
@@ -12,6 +12,43 @@ categories:
       docker_image: "bmscs"
       img: "img/bmscs.jpg"
 
+    - name: "BNR"
```

Comment:
For BNR the "local install instructions" are not correct. BNR is not a package and the function `startApplication()` does not exist.

![image](https://github.com/user-attachments/assets/6fe16cd6-7337-43f1-be9c-d3b1227195b5)

Correct install instrucions are part of the [README](https://github.com/Pandora-IsoMemo/BNR/blob/main/README.md) currently.

Either we should hide install instructions here or link to the README ...
Also the docker installation looks different here. But it should work also with our docker installation, I think ... :thinking: